src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/editprofile/AvatarComponentObject.java

```diff
@@ -26,6 +30,25 @@ public void uploadAvatar(String file) {
     Log.log("typeInFileToUploadPath", "type file " + file + " to upload it", true, driver);
   }
 
+  /**
+   * Select one of the default avatars we offer for users.
+   * @return URL of the selected avatar
+   */
+  public String pickRandomDefaultAvatar() {
+    WebElement chosenAvatar = defaultAvatarImages.stream()
+        .filter(e -> !e.getAttribute("data-name").equals("Avatar.jpg"))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the "Avatar.jpg" string literal on the left side of this string comparison. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1132)